xscen/extract.py

```diff
@@ -131,6 +131,7 @@ def extract_dataset(
     xr_open_kwargs: dict = None,
     xr_combine_kwargs: dict = None,
     preprocess: Callable = None,
+    resample_method: Optional[str] = None,
```

Comment:
I think this should be a dictionary, since the method might vary per variable.

Reply:
totaly right